doc/release-notes-27596.md

```diff
@@ -0,0 +1,7 @@
+Pruning
+-------
+
+When using assumeutxo with `-prune`, the prune budget may be exceeded. Prune budget is
```

Comment:
51622e9c1f7ed16f65844817a4a5e9bcbc157b4f

It might be helpful to be more precise here (and in the prune doc). This is what a user (not a dev) will need to know assuming they have limited memory, no?